api/tests/core/offerers/test_api.py

```diff
@@ -627,3 +627,25 @@ def move_venue_banner_to_legacy_location(venue, directory, timestamp):
     venue.bannerUrl = venue.bannerUrl.split("_")[0]
     os.rename(directory / f"{humanize(venue.id)}_{timestamp}", directory / f"{humanize(venue.id)}")
     os.rename(directory / f"{humanize(venue.id)}_{timestamp}.type", directory / f"{humanize(venue.id)}.type")
+
+
+class GetEligibleForSearchVenuesTest:
+    def test_without_max(self) -> None:
+        eligible_venues = offers_factories.VenueFactory.create_batch(3, isPermanent=True)
+        offers_factories.VirtualVenueFactory.create_batch(2)
```

Comment:
dans l'autre fichier de test, je trouvais ça plus explicite de préciser que les lieux permanents ne sont pas élligibles

```suggestion
        offers_factories.VenueFactory.create_batch(2, isPermanent=False)  # non-eligible venues
```